packages/block-library/src/gallery/use-image-sizes.js

```diff
@@ -35,17 +30,9 @@ export default function useImageSizes( images, isSelected, getSettings ) {
 				}
 
 				const sizes = imageSizes.reduce( ( currentSizes, size ) => {
-					const defaultUrl = get( img, [
-						'sizes',
-						size.slug,
-						'url',
-					] );
-					const mediaDetailsUrl = get( img, [
-						'media_details',
-						'sizes',
-						size.slug,
-						'source_url',
-					] );
+					const defaultUrl = img?.sizes?.[ size.slug ]?.url;
```

Comment:
Agreed, however, I left it in place just to be consistent with other areas where we retrieve the exact same data. I'm happy to remove it, if you feel strongly about it, though.

Reply:
I don't feel strongly about it and it's more a nit but I think it may cause confusion (as we access it directly on the line mentioned above). From a reviewing perspective it's not clear to me whether access is safe or not and ideally code should reflect that.

Reply:
I see what you mean. I've removed optional chaining here in f2fe8fa2128dad4465e8b8add474082154308b27